src/config.rs

```diff
@@ -7,15 +7,20 @@ use std::path::{Path, PathBuf};
 
 #[derive(Serialize, Deserialize)]
 pub struct Config {
+    #[serde(default)]
```

Comment:
I doubt this default is doing anything at all... is it?

Reply:
If all the settings are not listed in the config file, it can fail

Reply:
ahh right, cool then